community/sample-apps/gatsby-contentful-auth0/app/README.md

```diff
@@ -0,0 +1,28 @@
+# gatsby-contentful-auth0
```

Comment:
Is this readme same as the top-level one?

Reply:
The top level is more detailed with setup instructions for contentful, auth0 etc. This is more specific to running the gatsby app alone.